src/main/java/net/fabricmc/meta/utils/MinecraftSemverConverter.java

```diff
@@ -0,0 +1,130 @@
+package net.fabricmc.meta.utils;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class MinecraftSemverConverter {
+    // From Fabric Loader
```

Comment:
This this a direct copy/paste?

Reply:
There may have been some minor modifications

Reply:
Needs to be documented, as this is going to be a pain point to keep updated.

Reply:
Replaced to use loader directly